src/monitoring_tests/reference_solver_surplus_test.py

```diff
@@ -95,7 +105,7 @@ def compare_orders_surplus(
 
     def get_trade_alternative(
```

Comment:
If we use the solution itself for stuff, I would probably split this into two functions. One for computing the solution and one for extracting order information: `get_soltution_alternative(uid, auction_instance) -> solution` and `get_solution_alternative(uid, auction_instance, solution) -> Trade`. Otherwise we are mixing concepts a bit.

Reply:
Attempted to follow your suggestion. Let me know what you think.